README.md

```diff
@@ -191,7 +191,7 @@ They should be set in the `env` at the job level as shown in the [example](#matr
 To find the correct the environment variable name, see the [Bundler docs](https://bundler.io/man/bundle-config.1.html) or look at `.bundle/config` after running `bundle config --local KEY VALUE` locally.
 You might need to `"`-quote the environment variable name in YAML if it has unusual characters like `/`.
 
-To perform caching, this action will use `bundle config --local path $PWD/vendor/bundle`.  
```

Comment:
Could you undo such changes? The trailing spaces are intentional to force a line break when rendering Markdown.

Reply:
Sure, fixed!

I am not sure why I thought removing them was a good idea tho 😅 

Thanks for pointing that out 😃